ts/src/huobi.ts

```diff
@@ -5402,7 +5402,7 @@ export default class huobi extends Exchange {
             });
         }
         const sorted = this.sortBy (rates, 'timestamp');
-        return this.filterBySymbolSinceLimit (sorted, market['symbol'], since, limit);
+        return this.filterBySymbolSinceLimit (sorted, market['symbol'], since, limit, true);
```

Comment:
@samgermain am I missing something? using the master version (without this fix) I get the exact same order 
```
n huobi fetchFundingRateHistory ETH/USD:ETH undefined 5                                                        
2023-04-29T10:35:14.258Z
Node.js: v18.14.0
CCXT v3.0.76
huobi.fetchFundingRateHistory (ETH/USD:ETH, , 5)
2023-04-29T10:35:16.487Z iteration 0 passed in 434 ms

     symbol |           fundingRate |     timestamp |                 datetime
------------------------------------------------------------------------------
ETH/USD:ETH |                0.0001 | 1682208000000 | 2023-04-23T00:00:00.000Z
ETH/USD:ETH | -0.000423694800105871 | 1682236800000 | 2023-04-23T08:00:00.000Z
ETH/USD:ETH |                0.0001 | 1682265600000 | 2023-04-23T16:00:00.000Z
ETH/USD:ETH | -0.000224023949263782 | 1682294400000 | 2023-04-24T00:00:00.000Z
ETH/USD:ETH | -0.000540166457656147 | 1682323200000 | 2023-04-24T08:00:00.000Z
```

Reply:
That's not the same, those dates are 4-23 and 4-24, the most recent dates are 4-27 & 4-28 (4-28 and 4-29 now)